src/levels.js

```diff
@@ -1,11 +1,20 @@
-import _ from 'lodash';
+import reduce from 'lodash.reduce';
```

Comment:
why like that and not` import { reduce, inRange, isObject, ... } from 'lodash' `?